docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaEmailWebformHandler.php

```diff
@@ -74,6 +74,7 @@ public function getEmailMessage($foiaRequestId, WebformSubmissionInterface $webf
     // Get form submission contents.
     $submissionContents = $webformSubmission->getData();
     $this->listFileAttachmentNamesInSubmission($submissionContents);
+    $submissionContents['confirmation_id'] = $webformSubmission->id();
```

Comment:
This is better than what I did to add the request id. I did that within the `formatSubmissionContentsAsTable` method. Do you mind refactoring that to match the approach you took here?